extensions/BMO/template/en/default/pages/etiquette.html.tmpl

```diff
@@ -18,133 +18,149 @@
   #
   # Contributor(s): Stefan Seifert <nine@detonation.org>
   #                 Gervase Markham <gerv@gerv.net>
+  #                 Emma Humphries <ech@emmah.net>
   #%]
 
 [% PROCESS global/header.html.tmpl
    title = "Bugzilla Etiquette"
    style = "li { margin: 5px } .heading { font-weight: bold }" %]
 
 <p>
-  There's a number of <i lang="fr">faux pas</i> you can commit when using
-  [%+ terms.Bugzilla %]. At the very
-  least, these will make Mozilla contributors upset at you; if committed enough
-  times they will cause those contributors to demand the disabling of your
-  [%+ terms.Bugzilla %] account. So, ignore this advice at your peril.
+  It is our intention that [% terms.Bugzilla %] remain a useful tool for reporting 
+  and commenting on [% terms.bugs %], feature requests, and tasks for the Mozilla community.
+  <i>No single contributor's work outweighs the importance of civility and professionalism 
```

Comment:
Would "professionalism" be an intimidating word for volunteers? And civility, while correct, is also higher-level English. Perhaps we could replace these two words with something like "politeness and respectful behaviour"?